src/status_im/contexts/communities/channel_details/view.cljs

```diff
@@ -0,0 +1,183 @@
+       {:on-press                #(rf/dispatch [:chat.ui/show-profile public-key])
+        :allow-multiple-presses? true
+        :on-long-press           show-profile-actions
+        :accessory               {:type     :options
+                                  :on-press show-profile-actions}})
+     item]))
+
+(defn contacts-section-header
+  [{:keys [title]}]
+  [quo/divider-label {:tight? true} title])
+
+(defn contacts-section-footer
+  [_]
+  [rn/view {:style {:height 8}}])
+
+(defn actions
+  [chat-id cover-bg-color]
+  (let [latest-pin-text                      (rf/sub [:chats/last-pinned-message-text chat-id])
+        pins-count                           (rf/sub [:chats/pin-messages-count chat-id])
+        {:keys [muted muted-till chat-type]} (rf/sub [:chats/chat-by-id chat-id])
+        community-channel?                   (= constants/community-chat-type chat-type)
+        muted?                               (and muted (some? muted-till))
+        mute-chat-label                      (if community-channel? :t/mute-channel :t/mute-chat)
+        unmute-chat-label                    (if community-channel?
+                                               :t/unmute-channel
+                                               :t/unmute-chat)]
+    [quo/channel-actions
+     {:container-style {:margin-vertical 20 :padding-horizontal 20}
+      :actions         [{:accessibility-label :action-button-pinned
+                         :big?                true
+                         :label               (or latest-pin-text
+                                                  (i18n/label :t/no-pinned-messages))
+                         :customization-color cover-bg-color
+                         :icon                :i/pin
+                         :counter-value       pins-count
+                         :on-press            (fn []
+                                                (rf/dispatch [:pin-message/show-pins-bottom-sheet
+                                                              chat-id]))}
+                        {:accessibility-label :action-button-mute
+                         :label               (i18n/label (if muted
+                                                            unmute-chat-label
+                                                            mute-chat-label))
+                         :customization-color cover-bg-color
+                         :icon                (if muted? :i/activity-center :i/muted)
+                         :on-press            (fn []
+                                                (if muted?
+                                                  (actions/unmute-chat-action chat-id)
+                                                  (actions/mute-chat-action chat-id
+                                                                            chat-type
+                                                                            muted?)))}]}]))
+
+
+(defn channel-details
```

Comment:
we have in the guidelines set that we should generally name the export component `view` of a view.cljs file. I think it works well for consistency. 
Also then we know what to expect the uses are of this ns. Could you update this and then mark all the other components in this file as private? 🙏